lib/cli/options.js

```diff
@@ -158,7 +173,12 @@ const parse = (args = [], defaultValues = {}, ...configObjects) => {
 const loadRc = (args = {}) => {
   if (args.config !== false) {
     const config = args.config || findConfig();
-    return config ? loadConfig(config) : {};
+    const configObject = config ? loadConfig(config) : {};
+
+    // Set _configPath for use by parse()
+    configObject._configPath = config;
```

Comment:
[Refactor] I'm not a big fan of setting "magic" properties that are only used by specific other places. They get hard to keep track of and can accidentally end up being exposed to users. Which is what's happening here: `loadRc` is a `@public` function and thus part of the public API. https://mochajs.org/api/module-lib_cli#.loadRc

I think the root need for this code change is that the `config` value also needs to be available in `parse()`, right? If so: how about extracting the contents of this `if` into a private shared function, then utilizing that function in two places?

```ts
  if (args.config !== false) {
    return myNewFancyFunction(args).configObject;
  }
```

_(roughly that, but maybe with more clear naming?)_